cmd/gin-repod/repo.go

```diff
@@ -54,10 +54,17 @@ func (s *Server) createRepo(w http.ResponseWriter, r *http.Request) {
 
 	rid := store.RepoId{Owner: owner, Name: creat.Name}
 
-	_, ok := s.checkAccess(w, r, rid, store.NoAccess)
+	user, ok := s.checkAccess(w, r, rid, store.NoAccess)
 	if !ok {
 		return
 	}
+	// make sure routes user and token user are identical
+	if owner != user.Uid {
+		http.Error(w, "Invalid repository owner name", http.StatusBadRequest)
+		fmt.Fprintf(os.Stderr,
```

Comment:
`Server.log` was meant to be used for logging stuff if we have a server instance around. I guess it is not yet consistently used throughout the server code, but I think it should be.